src/app/header/header.component.html

```diff
@@ -10,6 +10,7 @@
                     'header-button-dark-mode': darkModeService.isDarkMode
                 }"
                 type="button"
+                title="hap menunë"
```

Comment:
A change to "hap menynë" has been asked from linguistics, let's swap it quickly.

Reply:
Sure !

Reply:
Feel free to merge after changing it, resolving conversation.